CometAnalyzer.java

```diff
@@ -32,1106 +32,950 @@
+                IJ.log(i + " centerline diff too big invalid");
             }
+            if (comet.status == Comet.VALID)
+                validCount++;
+        }
         return validCount;
     }
 
-    private boolean isOnEdge(ImageProcessor ip, Roi r){
+    private boolean isOnEdge(ImageProcessor ip, Roi r) {
         Rectangle br = r.getBounds();
 
-        if(br.x <= 1){
+        if (br.x <= 1) {
             return true;
-            }
-        if(br.y <= 1){
+        }
+        if (br.y <= 1) {
             return true;
-            }
-        if((br.x + br.width) >= (ip.getWidth()-1)){
+        }
+        if ((br.x + br.width) >= (ip.getWidth() - 1)) {
             return true;
-            }
-        if((br.y + br.height) >= (ip.getHeight()-1)){
+        }
+        if ((br.y + br.height) >= (ip.getHeight() - 1)) {
             return true;
-            }
-        return false;
         }
+        return false;
+    }
 
     // private void yMoments(ImageProcessor ip){
-        // double sumpix = 0.0;
-        // double sumy1 = 0.0;
-        // double sumy2 = 0.0;
-        // double sumy3 = 0.0;
-        // double sumy4 = 0.0;
-        // double xc, yc;
-        // double ymean, yvar;
-        // double yskew, ykurt;
-        // Rectangle roi = ip.getRoi();
-        // byte[] mask = ip.getMaskArray();
-        // Rectangle br = roi.getBounds();
-        // double pix;
-        // int cnt1=0;
-
-        // int counter = 0;
-        // for(int y=br.y; y<(br.y+br.height); y++){
-            // for(int x=br.x; x<(br.x+br.width); x++){
-                // if(mask[counter++]!=0){
-                    // pix = ip.getPixelValue(x,y);
-                    // xc = x+0.5;
-                    // yc = y+0.5;
-                    // sumpix += pix;
-                    // sumy1 += pix*yc;
-                    // cnt1++;
-                    // }
-                // }
-            // }
-        // ymean = sumy1/sumpix;
-
-        // IJ.log("Cnt all: "+counter);
-        // IJ.log("Cnt no mask: "+cnt1);
-
-        // counter = 0;
-        // for(int y=br.y; y<(br.y+br.height); y++){
-            // for(int x=br.x; x<(br.x+br.width); x++){
-                // if(mask[counter++]!=0){
-                    // pix = ip.getPixelValue(x,y);
-                    // xc = x+0.5;
-                    // yc = y+0.5;
-                    // sumpix += pix;
-                    // sumy2 += pix*Math.pow((yc-ymean),2);
-                    // sumy3 += pix*Math.pow((yc-ymean),3);
-                    // sumy4 += pix*Math.pow((yc-ymean),4);
-                    // }
-                // }
-            // }
-        // yvar = sumy2 / sumpix;
-        // yskew = sumy3 / (sumpix * Math.pow(yvar,1.5));
-        // ykurt = sumy4 / (sumpix * Math.pow(yvar,2)) - 3.0;
-        //IJ.log(ymean + "\t" + yvar + "\t" + yskew + "\t" + ykurt);
-        // }
-
-
-    private int getXIntensityCentroid(ImageProcessor ip){
+    // double sumpix = 0.0;

```

Comment:
In a number of places, commented out code is left but indentation (and sometimes new lines) are removed. Can these be reverted to keep the original?

Reply:
OK, so you want to keep the commented out code? Is there a reason? there may be a better way to store it is why I ask.

Reply:
It might be fine to remove commented out code but keeping it in a mangled form doesn't make sense. My suggestion is just to revert these changes and keep the original, that's the easiest option I think.